src/cf-base.less

```diff
@@ -246,6 +250,19 @@ h6,
     line-height: unit( @base-line-height-px / @font-size );
 }
 
+.subheader {
+    // For longer subheaders that we don't want to be too large or heavier than
+    // the regular weight.
+    @font-size: 18px;
+
+    margin-top: 0;
+    // Spacing in the design manual is based off of the baseline so this margin
+    // number has been eyeballed to be 30px.
+    margin-bottom: unit( 21px / @font-size, em );
```

Comment:
Happy to do it, but I'd like to do it everywhere in this file/project at once, and would prefer to tackle it in a later PR.